uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-work-patterns.js

```diff
@@ -145,6 +157,10 @@ define([
 
         return workPattern;
       });
+
+      vm.customWorkpattern = _.sortBy(vm.customWorkpattern, function (customWorkpattern) {
```

Comment:
if possible, chain (also, shouldn't the property be called `vm.customWorkPatterns`?)
```js
vm.customWorkPatterns = _.(vm.customWorkPatterns)
  .map(function (workPattern) {

  })
  .sortBy(function (workPattern) {

  })
  .value();
```